### PR TITLE
Fix indentation in benchmark script

### PR DESCRIPTION
**Pull request by adamjstewart**
Without this, the script errors out with:
```
Traceback (most recent call last):
  File "~/rtree/tests/off/benchmarks.py.off", line 134, in <module>
    t = timeit.Timer(
  File "~/lib/python3.10/timeit.py", line 123, in __init__
    compile(stmtprefix + stmt, dummy_src_name, "exec")
  File "<timeit-src>", line 3
    hits = [p.object for p in index.intersection(bbox, objects=insert_object)]
IndentationError: unexpected indent
```
Could also run style tools on these files if desired.

### CODE REVIEW

**Comment by mwtoews** [comment]
I've added a few more adjustments in subsequent commits.

File: MANIFEST.in [hunk 1]
```diff
@@ -1,4 +1,5 @@
 include *.md *.txt
 include MANIFEST.in
+recursive-include benchmarks *
```
Comment (adamjstewart) [inline MANIFEST.in:3]: Why include benchmark scripts in the sdist? It doesn't get installed right? Honestly I wouldn't even include tests/docs/txt either

Comment (mwtoews) [reply MANIFEST.in:3]: it's only in sdist, not in the build dist which only has a few files.

Comment (adamjstewart) [reply MANIFEST.in:3]: But what's the point of including these in the sdist?

Comment (adamjstewart) [reply MANIFEST.in:3]: Up to you. Personally I try to keep the sdist limited to the smallest possible collection of files needed to build the bdist and use the GitHub tarball as the authoritative collection of all files.

Comment (mwtoews) [reply MANIFEST.in:3]: I'll keep it in sdist for now as a matter of continuity, and that it's a really small file that someone may find useful.

**Pull request merged**